services/QuillLMS/app/models/activity_pack_sequence.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: activity_pack_sequences
+#
+#  id             :bigint           not null, primary key
+#  release_method :string           default("immediate")
```

Comment:
`release_method` is new since I viewed the RFC. Is the plan to create a record for all assigned activity packs (both staggered and immediate)? 

If one with `immediate` is created, are we creating `ActivityPackSequenceUnit`s for them as well?

Reply:
I was thinking about doing this on all, but I think to keep scope small we'll just create staggered ones for now.